R/calcHDDCDD.R

```diff
@@ -26,7 +28,23 @@ calcHDDCDD <- function(mappingFile,
                        bait = FALSE,
                        multiscen = FALSE,
                        rasDir = NULL,
-                       cacheDir = NULL) {
+                       cacheDir = NULL,
+                       fromSource = FALSE) {
+
+  if (isTRUE(fromSource)) {
+    # Read previously calculated data from source
+    hddcdd <- readSource("HDDCDDtemp")
+
+    weight <- hddcdd
+    weight[] <- 1
```

Comment:
I am not sure if I understand this correctly. Wouldn't this mean that regions with different population sizes are treated equally in regional aggregation? Why is the weight not population?

Reply:
That's the better weight for aggregation, I agree. But here we have already aggregated values that are disaggregated to ISO level. For this, the weight doesn't matter (because HDD and CDD are intensive) and I set it to 1. Now that I have the data at ISO level, I will aggregate it with population weights.